syntax = "proto3";

package orderexecutor;

option go_package = "orderexecutorpb";

message ExecuteOrderRequest {
}

message ExecuteOrderResponse {
  bool success = 1;
  string message = 2;
}

message PingRequest {
}

message PingResponse {
  string message = 1;
}

service OrderExecutorService {
  rpc ExecuteNextOrder(ExecuteOrderRequest) returns (ExecuteOrderResponse);
  rpc Ping(PingRequest) returns (PingResponse);
}
